ghcide/src/Development/IDE/Plugin/Completions.hs

```diff
@@ -77,10 +79,18 @@ produceCompletions = do
               let env = hscEnv sess
               -- We do this to be able to provide completions of items that are not restricted to the explicit list
               res <- liftIO $ tcRnImportDecls env (dropListFromImportDecl <$> imps)
-              case res of
-                  (_, Just rdrEnv) -> do
+              inScope <- liftIO $ tcRnImportDecls env imps
```

Comment:
Can we compute these two in parallel using something like https://hackage.haskell.org/package/async-2.2.2/docs/Control-Concurrent-Async.html#v:concurrently ?